packages/renderer/src/lib/learning-center/GuideCard.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+import type { Guide } from '../../../../main/src/plugin/learning-center/learning-center-api';
+import Button from '../ui/Button.svelte';
+export let guide: Guide;
+</script>
+
+<div class="flex flex-1 flex-col bg-charcoal-600 pb-4 rounded-lg hover:bg-zinc-700">
+  <dif class="pt-4">
+    <div class="px-4">
+      <img src="{`data:image/png;base64,${guide.icon}`}" alt="{guide.id}" style="{`height:48px;`}" />
```

Comment:
With height 48px the Lima logo spans the entire section, but the other logos look okay (docker, podman, etc.).

Reply:
That is the only way IMO to show logos that is not square, like Kind, Lima, NodeJS and etc.

Reply:
Updated guides.json and this is moot for now.